code/modules/reagents/chemistry/machinery/chem_master.dm

```diff
@@ -293,6 +460,51 @@
 	// Transfer modal information if there is one
 	data["modal"] = ui_modal_data(src)
 
+	data["maxnamelength"] = MAX_CUSTOM_NAME_LEN
```

Comment:
Can you change the data that uses defines to be a part of a ui_static_data call instead? Along with all the for loops with data that never changes